src/NewChain/AuxiliaryChain.ts

```diff
@@ -219,6 +219,25 @@ export default class AuxiliaryChain {
     return ostCoGateway.progressMint(messageHash, hashLockSecret, this.txOptions);
   }
 
+  /**
+   * Resets organization contracts admin address to 0x.
+   *
+   * @param organization Auxiliary chain organization address.
+   * @param txOptions Transaction options.
+   *
+   * @returns {Promise} Promise containing transaction receipt.
+   */
+  public async resetOrganizationAdmin(
+    organization,
+    txOptions,
+  ): Promise<Object> {
+    this.logInfo('reseting auxiliary chain organization admin.', { organization, txOptions } );
+    const contractInstance = new MosaicContracts(undefined, this.web3);
+    const tx = contractInstance.AuxiliaryOrganization(organization)
+          .methods.setAdmin('0x0000000000000000000000000000000000000000');
```

Comment:
Why do you create crazy constructs here instead of using the available `ContractInteract.Organization`? You probably don't have to import `MosaicContracts`. See for comparison line 209 ff.:

```js
  public async progressWithSecret(
    auxiliaryOstCoGatewayAddress: string,
    messageHash: string,
    hashLockSecret: string,
  ): Promise<void> {
    this.logInfo('progressing mint with secret');
    const ostCoGateway = new ContractInteract.EIP20CoGateway(
      this.web3,
      auxiliaryOstCoGatewayAddress,
    );
    return ostCoGateway.progressMint(messageHash, hashLockSecret, this.txOptions);
  }
```

Reply:
ContractInteract.Organization doesn't implement setAdmin function in mosaic.js 😿  . That's why used MosaicContracts.

Reply:
🤦‍♂ ok, sorry. Can you please add a comment here that that is the reason?

Reply:
Done!